src/Runtime.php

```diff
@@ -4,20 +4,30 @@
 
 abstract class Runtime
 {
+    public const NODEJS_20 = 'nodejs20.x';
+
+    public const NODEJS_18 = 'nodejs18.x';
+
     public const NODEJS_16 = 'nodejs16.x';
 
     public const NODEJS_14 = 'nodejs14.x';
 
-    public const NODEJS_12 = 'nodejs12.x';
+    public const PYTHON_311 = 'python3.11';
+
+    public const PYTHON_310 = 'python3.10';
 
     public const PYTHON_39 = 'python3.9';
 
     public const PYTHON_38 = 'python3.8';
 
     public const PYTHON_37 = 'python3.7';
 
+    public const RUBY_32 = 'ruby3.2';
+
     public const RUBY_27 = 'ruby2.7';
 
+    public const JAVA_17 = 'java17';
```

Comment:
Can add Java 21 as well here. 👍🏻
```suggestion
    public const JAVA_21 = 'java21';

    public const JAVA_17 = 'java17';
```

Reply:
Can you point me to when was Java 21 added? So I can update this PR

Reply:
https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md#214970